### ISSUE TITLE
 IM_ASSERT(i < Size); return Data[i]; throws error on TBF

### ISSUE THREAD

**Issue by Dj-jom2x** [Kaldaien/SpecialK#24]
imgui.h  IM_ASSERT(i < Size); return Data[i]; throws error in  line 903 ... it says its null when i try to do ... ctrl shift backspace ...

**Comment by Dj-jom2x** [comment]
im trying to make it latest special k .. it works best on me XD my laptop fan doesnt growl when i use the latest but the problem is when i press ctrl shit backspace .. it will show the menu then crash

and when i debug it this what it looks like 
![screenshot 38](https://cloud.githubusercontent.com/assets/12135498/25385054/422e3bba-29f5-11e7-90a9-619bcf4c70fe.png)

**Comment by Dj-jom2x** [comment]
hmm i try to debug it again .. now i altab fast so i can access the debuger ... i found the culprit 
// this always return null  this part g.IO.Fonts->Fonts[0];
static ImFont* GetDefaultFont()
{
    ImGuiContext& g = *GImGui;
    return g.IO.FontDefault ? g.IO.FontDefault : g.IO.Fonts->Fonts[0];
	 
}

**Comment by Kaldaien** [comment]
Are you trying to use the latest build of Special K with Tales of Berseria Fix?

I don't think that will actually work for a couple of reasons... the ABI has changed in SpecialK.dll since the last released version of TBF and font loading has changed slightly.

**Comment by Dj-jom2x** [comment]
yup sad :( my gpu usage is not 99% always on this build .. and my laptop is not growling ... i can still use the config.ini ... but ctrl + shift + backspace cant ...

when ImGui::NewFrame() is called

this will called  SetCurrentFont(GetDefaultFont());

but GetDefaultFont() this will throw g.IO.Fonts->Fonts[0] a null ...

**Comment by Kaldaien** [comment]
Try this:

[test.zip](https://github.com/Kaldaien/SpecialK/files/955340/test.zip)

That combination of DLLs is compatible on my system. I'm not ready to release an updated version of this project right now.

**Comment by Dj-jom2x** [comment]
wow it works .. <3 thanks

**Comment by Dj-jom2x** [comment]
the only problem is when i close it the mouse cursor is gone xD

**Comment by Kaldaien** [comment]
This should fix that problem:
[test2.zip](https://github.com/Kaldaien/SpecialK/files/955537/test2.zip)

The mouse cursor is the reason I haven't updated TBFix in a while, actually :) I completely overhauled the code to manage mouse cursors in Special K and it conflicted with my earlier work in TBFix. This should give cursor management responsibilities to Special K.

**Comment by Dj-jom2x** [comment]
i really dont undersntant the 60 fps but slow motion hahaha xD if i limit it to 30 fps it will fix but 60 fps xD ... 60fps animation are smooth xD their face is not blurry ...

slomo is so random xD i dont know what triggers them .. it has the same temp on 30 fps and 60 fps xD but still

**Comment by Dj-jom2x** [comment]
how about the mouse is always rotating the anime when I open the menu .. or if the game is pause

**Comment by Dj-jom2x** [comment]
just a  question do I really running at 60 fps under 30 fps xD the fpps on the left side it says 30 fps xD but on the right side is 104fps .. when I remove the fix its laggin ... but when I got the fix .. its smooth .. if I set the main game in 60 it get slowmo xD ... but if 30 it runs good with fix ..
